### PR TITLE
docs(root): add empty state documentation

### PR DESCRIPTION
**Pull request by GCHQ-Developer-530**
## Summary of the changes

Add empty state documentation with code examples. To test this, the components will have to be built locally from develop until the next release. 

## Related issue

 #487

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/content/structured/components/empty-state/accessibility.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/accessibility.mdx:6]:
```suggestion
title: "Empty state"
```

File: src/content/structured/components/empty-state/code.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/code.mdx:6]:
```suggestion
title: "Empty state"
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:8]:
```suggestion
title: "Empty state"
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:76]:
```suggestion
- No content yet exists due to the first-time use of the functionality.
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:77]:
```suggestion
- Data was not successfully loaded.
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:78]:
```suggestion
- A query or search term yielded no results.
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:90]: Remove? Repeat of above?

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:218]:
```suggestion
- [Alert](/components/alert).
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (MI6-94) [inline src/content/structured/components/empty-state/guidance.mdx:219]:
```suggestion
- [Loading indicator](/components/loading-indicator).
```

File: src/content/structured/components/empty-state/guidance.mdx
Comment (DBD324) [reply src/content/structured/components/empty-state/guidance.mdx]: Good catch

File: src/content/structured/components/empty-state/guidance.mdx
Comment (DBD324) [inline src/content/structured/components/empty-state/guidance.mdx:38]: Remove heading of introduction and replace with heading 'Component demo'

File: src/content/structured/components/empty-state/guidance.mdx
Comment (DBD324) [inline src/content/structured/components/empty-state/guidance.mdx:40]: remove as unnecessary

**Review by DBD324 — CHANGES_REQUESTED** [review]
A couple of changes requested. Need to wait for build before approval can be given. Need to see page to check sizing of imagery etc.

**Comment by DBD324** [comment]
When reviewing images noticed two required changes:
- Add warning status to the second image in the content section. 
- Reduce size of second image in when not to use section to a half width image. 

Otherwise looks good.

**Review by DBD324 — APPROVED** [review]
All comments have been actioned. Guidance is looking good to me.

**Review by MI6-255 — APPROVED** [review]

**Comment by MI6-255** [comment]
Approved, just need the build fixing

**Pull request merged**
